src/v/coproc/script_dispatcher.cc

```diff
@@ -221,9 +221,9 @@ script_dispatcher::enable_coprocessors(enable_copros_request req) {
     /// can register for input topics that don't yet exist
     if (!deregisters.empty()) {
         vlog(coproclog.error, "Immediately deregistering ids {}", deregisters);
+        auto on_stack = disable_copros_request({.ids = std::move(deregisters)});
         auto reply = co_await client->disable_coprocessors(
```

Comment:
can you create a ticket to record this fix along with a link to the gcc bug https://gcc.gnu.org/bugzilla/show_bug.cgi?id=95599

Reply:
https://github.com/vectorizedio/redpanda/issues/891 done